keras/feature_column/base_feature_layer.py

```diff
@@ -23,7 +23,7 @@
 
 import tensorflow.compat.v2 as tf
 
-import collections
+import collections.abc
```

Comment:
Surely you should also be updating the locations that access `collections` members in this file?

Reply:
It seems someone else has already updated the line in https://github.com/keras-team/keras/commit/5a15df56673aa5db7841c58a1565c0605f28c271 . I also made the change but I guess as part of the merge it got resolved by itself. I think this PR can be closed.